vsce/server/src/server.ts

```diff
@@ -520,6 +520,26 @@ async function getCodeActions(diagnostics: Diagnostic[], textDocument: TextDocum
 			suggestions.forEach(suggestion => {
 				codeActions.push(getQuickFix(diagnostic, labelPrefix + suggestion, range, suggestion, textDocument));
 			});
+		} else if (diagnostic.code === "RE0048") {
+			const title = "Add Reach program header.";
+			const newText: string = "'reach 0.1';\n\n";
```

Comment:
Ideally it will get 0.1 from the `VERSION` file

Reply:
It would only be able to do that in development, and not production, so maybe we keep some `version.ts` file in `vsce/server/src`?

Reply:
Or, we can maybe grab it from the error message, since the error message gives a suggested header, anyway.

![Screen Shot 2021-12-19 at 5 03 30 PM](https://user-images.githubusercontent.com/43425812/146697995-3893c784-df46-4303-92b0-07c9bba1cf96.png)

Reply:
Actually, after thinking about it a bit more, I think it really does make sense for the "Quick Fix", or, "auto-correct", to match whatever header Reach's compiler suggests in its error message.

We want the "auto-correct" to fix the compiler error: it doesn't matter what `../../VERSION` just so happened to be in the repository at the time we published the extension.

I am assuming that the compiler error message will have the correct Reach header.

Reply:
Good idea

Reply:
Thanks.

Reply:
Thanks.